apps/src/StudioApp.js

```diff
@@ -98,15 +98,26 @@ var codegen = require('./lib/tools/jsinterpreter/codegen');
 var copyrightStrings;
 
 /**
- * The minimum width of a playable whole blockly game.
+ * Store experiment parameters.
  */
+const isBigPlayspaceExperiment = experiments.isEnabledAllowingQueryString(
+  experiments.BIG_PLAYSPACE
+);
+const bigPlaySpacePadding = queryParams('bigPlayspacePadding') || 160;
```

Comment:
Would you need to reload the page to modify the padding here? If so, and that's undesirable, could we make this a function like `getMaxResizableVisualizationWidth` to fix that?

Reply:
For now, yes.  The idea is that we can try different values in different situations to see what works best, before we move this out of being an experiment.  I'm not quite sure how a function would change this?

Reply:
While we're experimenting, my original comment doesn't matter. It seems good for now!